plugins/GSdx/GSRendererHW.cpp

```diff
@@ -958,9 +959,9 @@ bool GSRendererHW::OI_GodOfWar2(GSTexture* rt, GSTexture* ds, GSTextureCache::So
 		TEX0.TBW = FBW;
 		TEX0.PSM = FPSM;
 
-		if(GSTextureCache::Target* ds = m_tc->LookupTarget(TEX0, m_width, m_height, GSTextureCache::DepthStencil, true))
+		if(GSTextureCache::Target* tmp_ds = m_tc->LookupTarget(TEX0, m_width, m_height, GSTextureCache::DepthStencil, true))
 		{
-			m_dev->ClearDepth(ds->m_texture, 0);
+			m_dev->ClearDepth(tmp_ds->m_texture, 0);
 		}
```

Comment:
I'd personally prefer if it gets changed to it's expansion, IIRC there were some people complaining gabest that some acronyms didn't make sense.